gradle/verification-metadata.xml

```diff
@@ -50,9 +55,9 @@
             <pgp value="19beab2d799c020f17c69126b16698a4adf4d638"/>
          </artifact>
       </component>
-      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.1.1">
-         <artifact name="test-retry-gradle-plugin-1.1.1.jar">
-            <sha256 value="3d5fc32376b92bd87f405ca649b8bab4333afc1e03954945f26ab657e0c1fedf" origin="Generated by Gradle because artifact wasn't signed"/>
+      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.1.8">
```

Comment:
The test retry plugin entry isn't necessary any more, since it is now signed. Please remove it.

Reply:
Removed.